polars/polars-time/src/chunkedarray/utf8/infer.rs

```diff
@@ -274,48 +260,40 @@ impl TryFrom<Pattern> for DatetimeInfer<i32> {
     fn try_from(value: Pattern) -> PolarsResult<Self> {
         match value {
             Pattern::DateDMY => Ok(DatetimeInfer {
-                pattern_with_offset: PatternWithOffset {
-                    pattern: Pattern::DateDMY,
-                    offset: None,
-                },
+                pattern: Pattern::DateDMY,
                 patterns: patterns::DATE_D_M_Y,
                 latest_fmt: patterns::DATE_D_M_Y[0],
                 transform: transform_date,
                 transform_bytes: StrpTimeState::default(),
                 fmt_len: 0,
                 logical_type: DataType::Date,
-                utc: false,
             }),
             Pattern::DateYMD => Ok(DatetimeInfer {
-                pattern_with_offset: PatternWithOffset {
-                    pattern: Pattern::DateYMD,
-                    offset: None,
-                },
+                pattern: Pattern::DateYMD,
                 patterns: patterns::DATE_Y_M_D,
                 latest_fmt: patterns::DATE_Y_M_D[0],
                 transform: transform_date,
                 transform_bytes: StrpTimeState::default(),
                 fmt_len: 0,
                 logical_type: DataType::Date,
-                utc: false,
             }),
             _ => polars_bail!(ComputeError: "could not convert pattern"),
         }
     }
 }
 
 impl<T: NativeType> DatetimeInfer<T> {
-    pub fn parse(&mut self, val: &str, offset: Option<FixedOffset>) -> Option<T> {
-        match (self.transform)(val, self.latest_fmt, offset, self.utc) {
+    pub fn parse(&mut self, val: &str) -> Option<T> {
+        match (self.transform)(val, self.latest_fmt) {
```

Comment:
most of this file is just removing the now-unnecessary `offset` and `utc` arguments, and updating the rest accordingly